code/modules/research/xenoarchaeology/artifact/artifact_replicator.dm

```diff
@@ -21,48 +21,105 @@
 	..()
 
 	var/list/viables = list(
+		//Medical
+		/obj/item/weapon/reagent_containers/glass/bottle/chloralhydrate,
 		/obj/item/roller,
-		/obj/structure/closet/crate,
-		/obj/structure/closet/acloset,
+		/obj/item/device/healthanalyzer,
+		/obj/item/weapon/reagent_containers/glass/bottle/chloralhydrate,
```

Comment:
Gee Billy your mom lets you have TWO bottles of chloral?

Reply:
FREE SHITTER JUICE

Reply:
gay

Reply:
replaced with normal toxin, shitter juice is boring anyway